Content.Server/GameTicking/GameTicker.Spawning.cs

```diff
@@ -237,30 +250,6 @@ private void SpawnPlayer(ICommonSession player,
             var jobName = _jobs.MindTryGetJobName(newMind);
             _admin.UpdatePlayerList(player);
 
-            if (lateJoin && !silent)
```

Comment:
It's better to  keep this code but add a && false to minimize line changes, then comment // RMC14 next to it